### PR TITLE
TD-2815 update card summary component

### PR DESCRIPTION
**Pull request by AronPetrauskas**
Closes [TD-2815](https://sce.myjetbrains.com/youtrack/issue/TD-2815) Update CardSummary component
Contributes to [TD-2784](https://sce.myjetbrains.com/youtrack/issue/TD-2784) Create Versions tab for road details

## Changes

Removes duplicate code by integrating `Infographic` into `SummaryCard`. Adds ability to display `VersionChip`.

- Replaced `CardMedia` with `Infographic` in `SummaryCard`
- Added ability to include `VersionChip`
- Added test to check `SummaryCard` renders with media and `VersionChip`
- Removed `mediaHeight` and `mediaWidth`, where these values were used for calculations they were replaced with absolute values.
- Replaced `<Fragment>` with `<Stack>` to be able to pass tag 'id' as per QA testing process.

## UI/UX
[Figma](https://www.figma.com/design/0CVK945S1cs1ku0sUXcgv7/VIRTO.SCENE?node-id=7668-33318&t=LwfGD5GH1wLu0vtX-0)

Figma screenshot:
image

Storybook screenshot:
image

## Testing notes

1. `pnpm i`
2. `pnpm run storybook` This should open storybook.
3. Use the sidebar to navigate to `CARDCONTENT/SummaryCard`
4. Using the sidebar navigate to "ScenarioExample"
5. Make sure the SummarCard renders with an image and `VersionChip`

## Author checklist before assigning a reviewer

- [x] Reviewed my own code-diff.
~- [ ] Branch has been run in docker.~
- [x] PR assigned to me or an appropriate delegate.
- [x] Relevant labels added to the PR.
- [x] Appropriate tests have been added.
- [x] Lint and test workflows pass.

### CODE REVIEW

**Review by m4manjesh — CHANGES_REQUESTED** [review]

**Comment by AronPetrauskas** [comment]
Added changed requested on PR review. I also sneaked in a quick fix that stops the `VersionChip` from overflowing the `Infographic` component.

File: src/Card/SummaryCard/SummaryCard.stories.tsx [hunk 1]
```diff
@@ -232,11 +230,12 @@ export const ScenarioExample = {
       </TableContainer>
     ),
     labels,
-    media: "https://picsum.photos/400/200",
+    media: "https://picsum.photos/336/190",
```
Comment (m4manjesh) [inline src/Card/SummaryCard/SummaryCard.stories.tsx:233]: Please add a story for verison with infographic.

File: src/CardContent/Infographic/Infographic.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ const Infographic = ({ media, version }: InfographicProps) => {
           width: 336
         }}
       >
-        <Box padding={1}>
+        <Box padding={1} sx={{ maxWidth: 320 }}>
```
Comment (m4manjesh) [inline src/CardContent/Infographic/Infographic.tsx:39]: please implement the same suggestion i requested on the other PR

**Review by m4manjesh — CHANGES_REQUESTED** [review]
Looks good, a small change to add

File: src/Card/SummaryCard/SummaryCard.stories.tsx (hunk 1 above)
Comment (AronPetrauskas) [reply src/Card/SummaryCard/SummaryCard.stories.tsx:233]: Line 238

File: src/CardContent/Infographic/Infographic.tsx (hunk 2 above)
Comment (AronPetrauskas) [reply src/CardContent/Infographic/Infographic.tsx:39]: Done on the other PR and pulled into this one.

File: src/Card/SummaryCard/SummaryCard.stories.tsx (hunk 1 above)
Comment (m4manjesh) [reply src/Card/SummaryCard/SummaryCard.stories.tsx:233]: I saw it in the Scenario Example, what i mean is a seperate story for WithVersionChip

**Review by m4manjesh — APPROVED** [review]
LGTM

**Pull request merged**